scripts/code_generation/jsoncan/src/can_database.py

```diff
@@ -288,7 +291,7 @@ class All:
     pass
 
 
-@dataclass(frozen=True)
```

Comment:
i would prefer to keep this around, is it possible to determine if a node is required to be fd before construction?

Reply:
it's nice to keep it around to prevent people for searching for mutations across the codebase